core/src/main/scala/com/metamx/tranquility/druid/DruidBeams.scala

```diff
@@ -857,7 +860,8 @@ object DruidBeams
     _alertMap: Option[Dict] = None,
     _objectWriter: Option[ObjectWriter[EventType]] = None,
     _timestamper: Option[Timestamper[EventType]] = None,
-    _taskContext: Option[Dict] = None
+    _taskContext: Option[Dict] = None,
+    _bitmapType: Option[String] = Some("concise")
```

Comment:
Minor: _bitmapType might just have String type as it has a non-None default value however it would be the only one such a field so it's up to you

Reply:
I was cautious to break a pattern initially and just in case left it as is.